packages/@aws-cdk/aws-glue/lib/table.ts

```diff
@@ -271,7 +271,11 @@ export class Table extends Resource implements ITable {
     });
 
     this.tableName = tableResource.tableName;
-    this.tableArn = `${this.database.databaseArn}/${this.tableName}`;
+    this.tableArn = this.node.stack.formatArn({
+      service: 'glue',
+      resource: 'table',
+      resourceName: Fn.join('', [this.database.databaseName, '/', this.tableName])
```

Comment:
This can be a string interpolation (what was previously in `tableArn`). Why the explicit `Fn.Join`? Does something break if we do it the other way?

Reply:
Didn't think to use string interpolation. Fixed now.